content/solutions/altinn-studio-repos/structure/_index.md

```diff
@@ -35,29 +35,42 @@ Under arbeid
 │  │  └── {{<icon name="fa-code">}} ...  
 │  │  
 │  ├── {{<icon name="fa-git-square">}} tjeneste-1  
-│  │  ├── {{<icon name="fa-folder">}} common ...      
+│  │  ├──  {{<icon name="fa-folder-open">}} Deployment ([helm]({{< ref "/technology/helm">}}))  
+│  │  │      ├── {{<icon name="fa-folder-open">}} templates  
+│  │  │      │     ├── {{<icon name="fa-code">}} _helpers.tpl  
+│  │  │      │     ├── {{<icon name="fa-code">}} deployment.yaml  
+│  │  │      │     ├── {{<icon name="fa-code">}} ingress.yaml  
+│  │  │      │     ├── {{<icon name="fa-code">}} NOTES  
+│  │  │      │     └── {{<icon name="fa-code">}} service.yaml  
+│  │  │      ├── {{<icon name="fa-code">}} .helmignore  
+│  │  │      ├── {{<icon name="fa-code">}} Chart.yaml  
+│  │  │      └── {{<icon name="fa-code">}} values.yaml  
 │  │  ├──  {{<icon name="fa-folder-open">}} Implementation  
 │  │  │   ├──  {{<icon name="fa-folder-open">}} Calculation   
-│  │  │   │   ├── {{<icon name="fa-code">}} [CalculationHandler.cs](calculationhandler)   
+│  │  │   │   └── {{<icon name="fa-code">}} [CalculationHandler.cs](calculationhandler)   
 │  │  │   ├── {{<icon name="fa-code">}} [InstantiationHandler.cs](instantiationhandler)  
 │  │  │   ├── {{<icon name="fa-code">}} [ServiceImplementation.cs](serviceimplementation)  
 │  │  │   └── {{<icon name="fa-code">}} [ValidationHandler.cs](validationhandler)  
+│  │  ├──  {{<icon name="fa-folder-open">}} Metadata  
+│  │  │       ├── {{<icon name="fa-code">}} [ServiceMetadata.json](servicemetadata)  
+│  │  │       └── {{<icon name="fa-code">}} workflow.json  
```

Comment:
when we have the workflow.bpmn down, do we still need this workflow.json file and the structure?

Reply:
The workflow.json file is still in use, I added an issue for removing it. When that is done, it can be removed from the documentation